galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.ceda.manager/src/main/java/dev/galasa/cicsts/ceda/internal/CedaManagerImpl.java

```diff
@@ -13,20 +13,51 @@
 import org.osgi.service.component.annotations.Component;
 
 import dev.galasa.ManagerException;
+import dev.galasa.cicsts.CedaManagerException;
 import dev.galasa.cicsts.CicstsManagerException;
+import dev.galasa.cicsts.CicstsManagerField;
 import dev.galasa.cicsts.ICeda;
 import dev.galasa.cicsts.ICicsRegion;
+import dev.galasa.cicsts.ceda.internal.properties.CedaPropertiesSingleton;
 import dev.galasa.cicsts.ceda.spi.ICedaManagerSpi;
 import dev.galasa.cicsts.spi.ICedaProvider;
 import dev.galasa.cicsts.spi.ICicstsManagerSpi;
 import dev.galasa.framework.spi.AbstractManager;
+import dev.galasa.framework.spi.AnnotatedField;
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.IFramework;
 import dev.galasa.framework.spi.IManager;
+import dev.galasa.framework.spi.language.GalasaTest;
 
 @Component(service = { IManager.class })
 public class CedaManagerImpl extends AbstractManager implements ICedaManagerSpi,ICedaProvider {
    
 	private ICicstsManagerSpi cicstsManager;
 	HashMap<ICicsRegion, ICeda> regionCeda = new HashMap<>();
+	
+	 protected static final String NAMESPACE = "cicsts";
+	    
+	    /* (non-Javadoc)
+	     * @see dev.galasa.framework.spi.AbstractManager#initialise(dev.galasa.framework.spi.IFramework, java.util.List, java.util.List, java.lang.Class)
+	     */
+	    @Override
+	    public void initialise(@NotNull IFramework framework, @NotNull List<IManager> allManagers, @NotNull List<IManager> activeManagers, @NotNull GalasaTest galasaTest) throws ManagerException {
+	        super.initialise(framework, allManagers, activeManagers, galasaTest);
+	        try {
+	            CedaPropertiesSingleton.setCps(framework.getConfigurationPropertyService(NAMESPACE));
+	        } catch (ConfigurationPropertyStoreException e) {
+	            throw new CedaManagerException("Unable to request framework services", e);
+	        }
+
+	        if(galasaTest.isJava()) {
+	           //*** Check to see if any of our annotations are present in the test class
+	            //*** If there is,  we need to activate
+	            List<AnnotatedField> ourFields = findAnnotatedFields(CicstsManagerField.class);
```

Comment:
No,  dont need the annotated fields check because there are no annotated fields for CEDA anymore.
just
if (java) {
youarerequired
}